### ISSUE TITLE
Original range of element 

### ISSUE THREAD

**Issue by ghost** [mrataj/BBCodeParser#1]
Hi. Thanks for sharing this library. I'm loving it right now.

I want to find the original range of an element. That is - where the element that was parsed was found in the original piece of text.

Say you have this:

NSString *string = @"AND I NEVER THOUGHT [quote author=iffets12345 game=iffets12345 ]Haha, EXACTLY![/quote]";

You parse it with the parser and it returns an element.

Now I want to replace the original location of the element with some new text based on what I have parsed out of the element.

Can this library do anything like that?

Basically - I want to turn that original string into something like this:

AND I NEVER THOUGHT "Haha, EXACTLY!" - iffets12345.
